docs/compatibility.md

```diff
@@ -353,7 +353,12 @@ the specified format string will fall into one of three categories:
 - Supported on GPU but may produce different results to Spark
 - Unsupported on GPU
 
-The formats which are supported on GPU and 100% compatible with Spark are :
+The formats which are supported on GPU and 100% compatible with Spark vary depending on the setting 
```

Comment:
But from reading below none of the formats are 100% compatible with Spark because even for `CORRECTED` and `EXCEPTION` we do not detect trailing characters.

Reply:
This was badly worded. We do claim that some of the formats are 100% compatible. I have made some changes to try and make this clearer.